### ISSUE TITLE
.vtk Dataset type: Rectilinear_grid vs Structured_points

### ISSUE THREAD

**Issue by remybt01** [elisepegg/py_bonemat_abaqus#1]
Hi,

My CT .vtk file uses the dataset type "STRUCTURED_POINTS" while the example .vtk file (example_ct_data.vtk) uses "RECTILINEAR_GRID". Rectilineargrid defines a point's locations with xyz data whereas Structuredpoints defines an origin and then spacing in xyz from that origin for each point.

data_import.py has a way to handle Rectilineargrid but not the Structuredpoints. The error I get is "Error reading VTK header, unrecognised format" from line 736 in data_import.py.

TL;DR Is there a way to change the code or is there a different library I can import to allow for the use of the STRUCTURED_POINTS data set type?

**Comment by elisepegg** [comment]
Hi remybt01,

Thanks for getting in touch. This is a limitation of the code, it is currently not able to read .vtk which is in the STRUCTURED_POINTS format. I have some plans to update the code soon, but i haven't yet done it and I don't expect to have time to do it very soon. 

I suggest that you try Bonemat (www.bonemat.org) - they have now incorporated my code into their software so that it can read abaqus input files, and they use a different data import code which is more versatile. If you decide to write a modification to data_import.py yourself then please feel free to do so and you can add this as a branch in github. 

Sorry I can't be of more help, but I am very grateful for you bringing this to my attention. I will add it to the list of changes I need to make to the code when I have some time.

Best wishes,

Elise
